hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/systemcontracts/hts/address_0x167/tokentype/TokenTypeCallTest.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.hedera.node.app.service.contract.impl.test.exec.systemcontracts.hts.tokentype;
+package com.hedera.node.app.service.contract.impl.test.exec.systemcontracts.hts.address_0x167.tokentype;
```

Comment:
Should the same package-transformation take place here in the unit tests?

Plus, no test that `staticCall` gets set right from the `0x167` translator vs the `0x16c` one.  (Suggest that such a test _not_ go through the `prepareHtsAttemptWithSelector...` helpers.)